sktime/forecasting/chronos.py

```diff
@@ -0,0 +1,161 @@
+
+
+# todo: add any necessary imports here
+
+
+class Chronos(BaseForecaster):
+    """Chronos forecaster by wrapping Amazon's Chronos model [1]_.
+
+    Direct interface to Amazon Chronos, using the sktime interface.
+    All hyperparameters are exposed via the constructor.
+
+    Parameters
+    ----------
+    modelName: str, required
+    top_p: float, default=1.0
+    top_k: int, default=50
+    temperature: float, default=1.0
+    num_samples: int, default=20
+    args_list: list, default=None
+    kwargs_dict: dict, default=None
+
+    References
+    ----------
+    . [1] https://github.com/amazon-science/chronos-forecasting
+
+    Examples
+    --------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.forecasting.chronos import Chronos
+    >>> from sktime.split import temporal_train_test_split
+    >>> from sktime.performance_metrics.forecasting import mean_absolute_percentage_error
+    >>> from sktime.forecasting.base import ForecastingHorizon
+    >>> import torch
+    >>> y = load_airline()
+    >>> y_train, y_test = temporal_train_test_split(y)
+    >>> fh = ForecastingHorizon(y_test.index, is_relative=False)
+    >>> forecaster = Chronos("amazon/chronos-t5-small", kwargs_dict={"torch_dtype": torch.bfloat16})
+    >>> forecaster.fit(y_train)
+    >>> y_pred = forecaster.predict(fh)
+    >>> mean_absolute_percentage_error(y_test, y_pred)
+    """
+
+    # tag values are "safe defaults" which can usually be left as-is
+    _tags = {
+        "python_dependencies": ["torch"],
+        "y_inner_mtype": "pd.Series",
+        "X_inner_mtype": "pd.DataFrame",
+        "scitype:y": "univariate",
+        "ignores-exogeneous-X": True,
+        "requires-fh-in-fit": False,
+        "authors": ["RigvedManoj"],
+        "maintainers": ["RigvedManoj"],
+    }
+
+    def __init__(self, modelName, num_samples=20, temperature=1.0, top_k=50, top_p=1.0, args_list=None, kwargs_dict=None):
+        self.modelName = modelName
+        self.num_samples = num_samples
+        self.temperature = temperature
+        self.top_k = top_k
+        self.top_p = top_p
+        self.args_list = args_list
+        self.kwargs_dict = kwargs_dict
+
+        super().__init__()
+
+    def _fit(self, y, X=None, fh=None):
+        """Fit forecaster to training data.
+
+        private _fit containing the core logic, called from fit
+
+        Writes to self:
+            Sets fitted model attributes ending in "_".
+
+        Parameters
+        ----------
+        y : pd.Series
+            Target time series to which to fit the forecaster.
+        fh : guaranteed to be ForecastingHorizon or None, optional (default=None)
+            The forecasting horizon with the steps ahead to predict.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables are ignored.
+
+        Returns
+        -------
+        self : reference to self
+        """
+        import torch
+        from chronos import ChronosPipeline
```

Comment:
My guess is as of now this is going to fail in CI, as there's no installation of this library. You need to add it in pyproject.toml and in dependency tag above. 

Since changes in pyproject.toml triggers a lot of jobs, I'll request you to consider commenting unnecessary jobs in CI file if possible. We'll revert this commenting part later of course.